src/pcre2_compile.c

```diff
@@ -5473,8 +5219,7 @@ Returns:        the number of < 256 characters added
 
 static unsigned int
 add_to_class_internal(uint8_t *classbits, PCRE2_UCHAR **uchardptr,
-  uint32_t options, uint32_t xoptions, compile_block *cb, uint32_t start,
-  uint32_t end)
+  uint32_t options, compile_block *cb, uint32_t start, uint32_t end)
```

Comment:
The reason of removing `xoptions` is that the only flag which is used is `PCRE2_EXTRA_CASELESS_RESTRICT`, and that flag does not change, so `cb->cx->extra_options` is good enough to access it.